src/components/NewWorkspaceModal.ts

```diff
@@ -319,11 +319,19 @@ const NewWorkspaceModal = withDisplayName(
     };
 
     const isBillingProjectApplicable = (project: BillingProject): boolean => {
-      // As of AJ-1164, if requireEnhancedBucketLogging is true, then azure billing projects are ineligible.
```

Comment:
My understanding of this ticket is that it removes the restriction imposed by AJ-1164 which disallowed import of data with `requireEnhancedBucketLogging ` into a new Azure workspace.